pkg/agent/nplagent/bootstrap/npl_agent_init.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
```

Comment:
I feel like there is no need for a `bootstrap` package here, these files can go in `pkg/agent/nplagent/ directly